### DISCUSSION TITLE
Socket Connection Error -- Dropping every 5 minutes

### DISCUSSION THREAD

**Discussion by nkreiger** [supabase/supabase#13694] [Questions]
I'm running a self-hosted instance of realtime in kubernetes. It appears to drop a websocket connection at exactly 5 minutes, consistently. I can't figure out if this is set somewhere, or why the connection is dropping. Has anyone experienced this?

**Comment by GaryAustin1** [comment]
This is mainly being worked here https://discord.com/channels/839993398554656828/1006358244786196510/threads/1095875854216724560
And seems like a time out issue with the environment running the realtime.
